.cursor/prompts/코드스타일_가이드라인.md

```
@@ -0,0 +1,1213 @@
+      headers: { "Content-Type": "multipart/form-data" },
+    });
+
+    // 로그아웃시 토큰 제거
+    sessionStorage.removeItem("accessToken");
+  },
+};
+```
+
+#### Command/Dto 타입 사용법
+
+**Command 객체 (요청시)**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)